application/pr/viewer.jsx

```diff
@@ -108,7 +144,7 @@ export default class PRViewer extends React.Component {
                 }
                 action = <span>{revert}{ revert && merge ? " — or " : (merge ? " — " : "")}{merge}</span>;
             } else {
-                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button></span>;
+                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button> { this.props.isAdmin || st.teamcontact ? <button name="nplc" onClick={this.askNPLC.bind(this)}>Ask for non-participant commitment</button> : "" }</span>;
```

Comment:
rather than `<button onClick>` I was thinking of putting the `<a>` currently generated in `askNPLC` directly in there (hence the `className=button`) - sorry I was unclear about this

Reply:
Oh sorry, my method is a bit overkill :)
Updated in addressed https://github.com/w3c/ash-nazg/pull/163/commits/3491667e19f54f1d4a3e43e40c05b151d66203a0